packages/fxa-content-server/server/lib/routes/react-app/index.js

```diff
@@ -82,6 +82,8 @@ const getReactRouteGroups = (showReactApp, reactRoute) => {
         'signin_unblock',
         'force_auth',
         'signin_recovery_code',
+        'signin_recovery_choice',
+        'signin_recovery_phone',
```

Comment:
You probably already know this but just to be clear / for others too: we only need to add routes to content-server this way if it's not behind `/settings/*` - content-server already points all settings routes to our React app if users hit them directly. Otherwise, without these content-server modifications, the server doesn't know what to do on a refresh of that page and it ends up blank, so this is probably preferred for routes not behind settings.

(I wanted to think that through myself too, so, 👍)

Reply:
Thank you, I wasn't 💯 sure on this so appreciate the confirmation!